.circleci/tests/erigon-all.json

```diff
@@ -1,9 +1,5 @@
 {
   "participants": [
-    {
```

Comment:
lets set the following to true in the `.circle/config.yml` file so that all of these tests run too; in the next PR we can make them nightly again

```
parameters:
  should-enable-check-latest-version-workflow:
    type: boolean
    default: true
```